anta/tests/system.py

```diff
@@ -102,19 +93,9 @@ class VerifyReloadCause(AntaTest):
     class Input(AntaTest.Input):
         """Input model for the VerifyReloadCause test."""
 
-        allowed_causes: list[str] = Field(default=["USER", "FPGA"])
+        allowed_causes: list[ReloadCause] = Field(default=["Reload requested by the user.", "Reload requested after FPGA upgrade"])
```

Comment:
I would expect this now
```suggestion
        allowed_causes: list[ReloadCause] = Field(default=["USER", "FPGA"])
```

Reply:
Hi, @gmuloc 
If a default value is set, the custom type validator will not be triggered, and as a result, the value won't be converted into the expected description.